### ISSUE TITLE
NTR: Mercury high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8436]
New term request Mercury high in blood (segal_220303184636):   UMLS: 1168068

**Comment by pnrobinson** [comment]
@MickeySegal


I am not sure that "High in blood" is what we want since there is no abnormal low level.
I would place this here [Blood xenobiotic HP:0410172 ](https://hpo.jax.org/app/browse/term/HP:0410172)
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4724159/

**Comment by pnrobinson** [comment]
@MickeySegal
after some thought, the following appeared to be best

Adding new term: 
- ID: HP:6000443
- label: Elevated blood mercury concentration
- parents: [Blood xenobiotic (HP:0410172)](https://hpo.jax.org/app/browse/term/HP:0410172)
- pmid: PMID:26644501
- def: The concentration of mercury in the blood is above the upper limit of a guidance value for total blood mercury concentrations that implies possible injurious effects.
- comment: Mercury testing may be considered in cases of high consumption of large predatory fish or exposure to mercury vapor .
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8437
- synonyms:
